netbsd/NetBSDProcessList.c

```diff
@@ -49,23 +50,67 @@ static char const *freqSysctls[] = {
    NULL
 };
 
+static void NetBSDProcessList_updateCPUcount(ProcessList* super) {
+   NetBSDProcessList* opl = (NetBSDProcessList*) super;
+   // https://nxr.netbsd.org/xref/src/sys/sys/sysctl.h#813
+   // Number of cpus, used with sysctl(3).
+   const int nmib[] = { CTL_HW, HW_NCPU };
+   // Number CPUs online, used with sysctl(3)
+   const int mib[] = { CTL_HW, HW_NCPUONLINE };
```

Comment:
What about?
```suggestion
   // Definitions for sysctl(3), cf. https://nxr.netbsd.org/xref/src/sys/sys/sysctl.h#813
   const int mib_ncpu_existing[] = { CTL_HW, HW_NCPU }; // Number of existing CPUs
   const int mib_ncpu_online[] = { CTL_HW, HW_NCPUONLINE }; // Number of online/active CPUs
```
Related renames appropriately below.

Reply:
This should also be done in the latest commit.